content/redirects/interested.json

```diff
@@ -0,0 +1,4 @@
+{
+  "from": "/interested",
```

Comment:
I would make the url more descriptive like this /committee-interest-form

Reply:
Unless `/interested` has already been reserved, I'd go with the shorter option. The idea is that it should be quick to type in